python/python/glide/routes.py

```diff
@@ -48,6 +48,24 @@ def __init__(self, slot_type: SlotType, slot_id: int) -> None:
         self.slot_id = slot_id
 
 
+class ByAddressRoute(Route):
+    def __init__(self, host: str, port: Optional[int] = None) -> None:
+        """Routes a request to a node by its DNS address
+
+        Args:
+            host (str): The DNS name of the node. If `port` is not provided, should be in the `{DNS}:{port}` format.
```

Comment:
same regarding DNS

Reply:
same as my comment above - should we support IP host addresses also?